lib/floki/selector/attribute_selector.ex

```diff
@@ -63,7 +63,7 @@ defmodule Floki.Selector.AttributeSelector do
     s.attribute
     |> get_value(attributes)
     # Splits by whitespaces ("a  b c" -> ["a", "b", "c"])
-    |> String.split(~r/\s+/)
+    |> String.split([" ", "\t", "\n"])
```

Comment:
Here as well. I think we need to consider the same problems here.

Reply:
fa64b01261028d47eb5d41c1f6aab840a781f636